traffic_ops/app/db/admin.go

```diff
@@ -303,12 +303,12 @@ func loadSchema() {
 }
 
 func reverseSchema() {
-	fmt.Println("WARNING: the '" + CmdReverseSchema + "' command will be removed with Traffic Ops Perl because it will no longer be necessary")
-	cmd := exec.Command("db/admin.pl", "--env="+Environment, CmdReverseSchema)
+	fmt.Fprintf(os.Stderr, "WARNING: the '%s' command will be removed with Traffic Ops Perl because it will no longer be necessary\n", CmdReverseSchema)
+	cmd := exec.Command("db/reverse_schema.pl")
```

Comment:
This command needs the MOJO_MODE environment variable injected in order to reverse_schema on a specific environment (rather than just "development"):
```
cmd := exec.Command("db/reverse_schema.pl")
cmd.Env = append(os.Environ(), "MOJO_MODE="+Environment)
...
```

With that fixed I believe this will be good to go.

Reply:
Really? Wouldn't it inherit the environment of the parent process? It seemed to be working for me, but if you think it needs this I'll add it.

Reply:
From the Go docs:
```go
        // If Env is nil, the new process uses the current process's
        // environment.
```
However, it looks like the re-write to Go dropped the Perl script's behaviour of setting `MOJO_MODE` - probably because only `reverse_schema` used it and that was calling back to the Perl script anyway.

tl;dr I took your suggestion.